scripting/cwx.sp

```diff
@@ -431,18 +432,17 @@ MRESReturn OnGetLoadoutItemPost(int client, Handle hReturn, Handle hParams) {
 	if (loadoutSlot < 0 || loadoutSlot >= NUM_ITEMS) {
 		return MRES_Ignored;
 	}
+
+	if (g_CurrentLoadout[client][playerClass][loadoutSlot].IsEmpty()) {
+		// we don't have nor want a custom item; let the game process it
+		return MRES_Ignored;
+	}
 	
+	// the loadout entity we keep track of isn't valid, so we may need to make one
+	// we expect to have to equip something new at this point
 	int storedItem = g_CurrentLoadout[client][playerClass][loadoutSlot].entity;
 	if (!IsValidEntity(storedItem) || GetEntityFlags(storedItem) & FL_KILLME
 			|| !HasEntProp(storedItem, Prop_Send, "m_Item")) {
-		// the loadout entity we keep track of isn't valid, so we may need to make one
```

Comment:
I feel like there's a reason this wasn't hoisted.  Can't think of it at the moment.